@Alyx/getFile.m

```diff
@@ -1,11 +1,20 @@
 function [fullPath, exists] = getFile(obj, eid, type)
-%GETFILE Recovers the full filepath(s) associated with a given eid
+%GETFILE Returns the full filepath for a given eid.
 %	Returns the full path of associated data files given a dataset or file
-%	record URL/eid.  Also returns logical array indicating whether files
-%	exist.
+%	record URL or eid (experiment ID). Also returns a logical array 
+% indicating whether files exist for each element in the `eid` array.
 %
-%   eid: full URL or eid of a dataset/file record on Alyx, or array thereof
-%   type: string indicating type of eid ('dataset' (default) or 'file')
+% Inputs:
+%   `eid`: a string or char array containing the full URL(s) or eid(s) of 
+%   dataset(s) / file record(s) on Alyx.
+%   `type`: a string indicating type of eid(s) ('dataset' (default) or 
+%   'file')
+%
+% Outputs:
+%   `fullPath`: a string array containing the full data file path(s) for
```

Comment:
I would change this to 'cellstring' as a string array is never returned by this function

Reply:
sure